packages/xxd_standalone.rb

```diff
@@ -1,24 +1,27 @@
 require 'package'
+Package.load_package("#{__dir__}/vim_runtime.rb")
 
 class Xxd_standalone < Package
   description 'Hexdump utility from vim'
   homepage 'https://www.vim.org/'
-  version '9.1.0969'
+  version Vim_runtime.version
```

Comment:
Should we put a comment to remind us this package needs rebuilt whenever vim_runtime is updated?

Reply:
We need to modify the updater logic to handle when versions have changed for stuff like this.

`crew update` already handles that... so we need to expand that to our updater.

Leaving that as work to be done.